aspnetcore/fundamentals/openapi/buildtime-openapi.md

```diff
@@ -36,65 +37,99 @@ Run the following command in the directory that contains the project file:
 ```dotnetcli
 dotnet add package Microsoft.Extensions.ApiDescription.Server --prerelease
 ```
+
 ---
 
-Upon installation, this package will automatically generate the Open API document(s) associated with the application during build and populate them into the application's output directory.
+The `Microsoft.Extensions.ApiDescription.Server` package automatically generates the Open API document(s) associated with the app during build and places them in the app's `obj` directory:
```

Comment:
Is obj right? I would have thought it would be in the output directory (or should be) so be within bin.

Reply:
run /TestScripts/create.cmd
what version of RC1 do you have?
I have 9.0.100-rc.1.24415.1

Reply:
My testing also shows that the default location is the obj directory, but you can change that (and I always do) with this setting in the project file:

```
  <PropertyGroup>
    <OpenApiDocumentsDirectory>$(MSBuildProjectDirectory)</OpenApiDocumentsDirectory>
  </PropertyGroup>
```